diff --git a/node_modules/rimble-ui/dist/es/Input/index.js b/node_modules/rimble-ui/dist/es/Input/index.js
index be8b4b7..e5e3042 100644
--- a/node_modules/rimble-ui/dist/es/Input/index.js
+++ b/node_modules/rimble-ui/dist/es/Input/index.js
@@ -10,7 +10,7 @@ import FileInput from './FileInput';
 import { StyledBox } from '../Box';
 var StyledInput = /*#__PURE__*/styled(StyledBox).attrs(function (props) {
   return {
-    as: 'input'
+    as: props.as
   };
 }).withConfig({
   displayName: "Input__StyledInput",
@@ -22,6 +22,7 @@ var StyledIconWrapper = /*#__PURE__*/styled.div.withConfig({
 })(["&{position:relative;display:flex;align-items:center;width:100%;}> ", "{padding-right:3rem;text-overflow:ellipsis;white-space:no-wrap;}> svg{position:absolute;right:1rem;visibility:hidden;}"], StyledInput);
 var WithValidationStyle = React.forwardRef(function (props, ref) {
   return React.createElement(StyledIconWrapper, null, React.createElement(StyledInput, _extends({}, props, {
+    as: 'input',
     ref: ref
   })), React.createElement(CheckCircle, {
     className: 'icon-valid'
@@ -32,10 +33,12 @@ var WithValidationStyle = React.forwardRef(function (props, ref) {
 var Input = React.forwardRef(function (props, ref) {
   if (props.type === 'file') {
     return React.createElement(FileInput, _extends({}, props, {
+      as: 'input',
       ref: ref
     }));
   } else {
     return React.createElement(StyledInput, _extends({}, props, {
+      as: 'input',
       ref: ref
     }));
   }
